From 5049e0b3c74a3a02c4436dd1902c504663ddc123 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Tue, 21 Oct 2014 16:58:42 +0800
Subject: [PATCH] hdmi:         Temporary diable YUV output. For some sink may
 not recognize         yuv format correctly, then picture will mask a pink
 layer.         Especially monitor device.

---
 drivers/video/rockchip/hdmi/rk_hdmi_lcdc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/hdmi/rk_hdmi_lcdc.c b/drivers/video/rockchip/hdmi/rk_hdmi_lcdc.c
index aa6f9ffbc83d..649a9909d7f0 100755
--- a/drivers/video/rockchip/hdmi/rk_hdmi_lcdc.c
+++ b/drivers/video/rockchip/hdmi/rk_hdmi_lcdc.c
@@ -624,13 +624,13 @@ int hdmi_init_video_para(struct hdmi *hdmi_drv, struct hdmi_video_para *video)
 		video->color_depth = HDMI_COLOR_DEPTH_10BIT;
 	else
 		video->color_depth = HDMI_COLOR_DEPTH_8BIT;
-
+/*
 	if (hdmi_drv->edid.ycbcr444)
 		video->output_color = VIDEO_OUTPUT_YCBCR444;
 	else if (hdmi_drv->edid.ycbcr422)
 		video->output_color = VIDEO_OUTPUT_YCBCR422;
 	else
-		video->output_color = VIDEO_OUTPUT_RGB444;
+*/		video->output_color = VIDEO_OUTPUT_RGB444;
 
 	/*For DVI, output RGB */
 	if (hdmi_drv->edid.sink_hdmi == 0)
-- 
2.35.3

